tests/scripts/all.sh

```diff
@@ -59,6 +59,15 @@
 # This script must be invoked from the toplevel directory of a git
 # working copy of Mbed TLS.
 #
+# The behavior on an error depends on whether --keep-going (alias -k)
+# is in effect.
+#  * Without --keep-going: the script stops on the first error without
+#    cleaning up. This lets you work in the configuration of the failing
+#    component.
+#  * With --keep-going: the script runs all requested components and
```

Comment:
perhaps it would be useful to split the 'always cleans up on exit' behavior into separate flag. The name "keep-going" does not imply the working environment is cleaned upon exiting.

Reply:
In keep-going mode, after a failure, the script keeps going. This includes running the cleanup code.

There could be a mode where the scripts stops at the first error, but does cleanup. That's out of scope and I don't remember anyone particularly wanting that.